Downloads.html

```diff
@@ -52,7 +53,7 @@
                 </div>
                 <div class="col-md-4">
                     <h2>Quick Start Guide</h2>
-                    <p>The DDF install guide and documentation are located on the wiki site.</p>
+                    <p>Get a sample or test installation up quickly.</p>

```

Comment:
Oh it's almost like a title description. Maybe this is better: `Get an installation up quickly for sample or test.`